### ISSUE TITLE
The access token provided is invalid

### ISSUE THREAD

**Issue by chrisjohnson** [mileswd/mac2imgur#113]
I have been getting this regularly for the last few months. Removing and reinstalling mac2imgur seems to fix it temporarily but then it comes back, and once it happens, it's on every upload. I can upload to imgur.com just fine. I know the API key stuff changed a while back but it seems like it's not fully fixed yet

**Comment by chrisjohnson** [comment]
<img width="360" alt="screen shot 2017-01-26 at 2 22 11 pm" src="https://cloud.githubusercontent.com/assets/178144/22346740/2868d202-e3d3-11e6-8f89-7e2ac1788014.png">

**Comment by jt3k** [comment]
I get the following message when I mac2imgur on two computers at the same time.

when I sign-in on the first computer, the second stops working.
And when I sign-in on the second, the first computer also stops working..

I don't know how to solve this problem.

**Comment by mp607** [comment]
mac2imgur on my only one mac. This problem +1

**Comment by chrisjohnson** [comment]
I've only got this installed on one mac as well, and I don't use any other imgur API software whatsoever. So this is definitely not a multi-mac or multi-login issue, at least not in my case.

**Comment by mcescalante** [comment]
I have had this issue pop up 2 times in the last week. Both times I fixed it by logging out and back in, but didn't investigate further

**Comment by mlebkowski** [comment]
Yeah, using the same account on two computers is a PITA. Can I manually transfer the token from one machine to another as a temporary fix? Is there any option to store the token on dropbox?

**Comment by bhargavjoshi** [comment]
This issue happens for me as well,

I need to sign out of the account and the sign in again, and it solves the problem. 

So it looks like token is getting expired.

It would be nice that if token is expired the app logs us out and redirect to the login page.

**Comment by kristynrb** [comment]
I'm getting this same issue. Sometimes quitting and opening solves it. Sometimes logging out and then logging back in helps. Right now, I can't get it to work at all. I keep receiving the same message that "The access token provided is invalid".

**Comment by benoitemile** [comment]
hi, I have this issue too, every few days, I have to logout/login; could you please fix ? Is there any time-related-permission when the token is granted ?

**Comment by chrisjohnson** [comment]
Issue still happening regularly here, the app is basically not usable sadly

**Comment by luckman212** [comment]
Same problem here. Anyone got a way to fix / debug this?
